### ISSUE TITLE
Change default component extension

### ISSUE THREAD

**Issue by tim-phillips** [mantrajs/mantra-cli#104]
It appears the `.jsx` extension is hard coded in:

https://github.com/mantrajs/mantra-cli/blob/255405efd4a680498b2e03df16fdb82b7bb165fb/lib/generators/utils.js#L26

Are there any plans to enable the user to generate components with a `.js` extension, like in the mantra sample blog app?

**Comment by sungwoncho** [comment]
Yeah. this is doable. [Here](https://github.com/mantrajs/mantra-cli/blob/master/lib/generators/utils.js#L24-L34) is how we are getting the extensions currently.

We can make a function that takes user config (read from `mantra-cli.yml`) and entity type (action, components, etc.) and conditionally outputs extensions. PR is welcomed.

**Comment by tim-phillips** [comment]
Great! I'll take a crack at it. I'll let you know here if I have any questions.
